docs/getting-started/quickstart.md

```diff
@@ -89,10 +90,9 @@ This creates a folder for your application, configures its layout, and installs
 If successful, it should print `Application initialized successfully!`.
 By default, `init` instantiates a "Hello, Database!" application which greets users and tracks the count of greetings per user.
 
+## Running Locally
 
-### Getting Started
-
-Before you can launch your app, you need a database.
+Before you can run your app, you need a database.
```

Comment:
At the end of this section (or at the beginning), I would be clear you can run the SDK anywhere.